docs/about/participating.rst

```diff
@@ -21,20 +21,20 @@ CIViC Principles
 
 How to Contribute
 ~~~~~~~~~~~~~~~~~
-There are several ways you can make a contribution to this important problem:
+There are several ways you can make a contribution to this important cause:
 
 
-- **Add:** Add evidence statements that support clinical actions associated with genomic variants (e.g. single nucleotide substitution, structural variant, gene fusion, etc.), add summaries of the corpus of evidence for a variant, suggest publications that should be curated, or help create a synthesized interpretation (i.e., Assertion) of the evidence supporting a variant in the context of a disease and its clinical implications.
+- **View:** Make use of the community-created content in your own research by browsing, searching, and examining detailed evidence items. All CIViC data and source code are provided freely for almost any use.
```

Comment:
This does make me wonder if "any" is even applicable here.

Reply:
Yeah, I wonder if we could end the sentence at freely. 
```suggestion
- **View:** Make use of the community-created content in your own research by browsing, searching, and examining detailed evidence items. All CIViC data and source code are provided freely.
```
```suggestion
- **View:** Make use of the community-created content in your own research by browsing, searching, and examining detailed evidence items. All CIViC data and source code are provided freely.
```